tests/components/sun/test_init.py

```diff
@@ -142,6 +150,8 @@
         state.attributes[sun.STATE_ATTR_NEXT_SETTING]) == \
         datetime(2016, 7, 26, 22, 19, 1, tzinfo=dt_util.UTC)
 
+    assert state.state == sun.STATE_ABOVE_HORIZON
```

Comment:
This doesn't really make sense. The sun is actually in the sky, but in the above code it rises (again) on 25/Jul, before setting on the 26/Jul. There's an issue with the astral calculator, but I'm not sure what it is exactly.